src/NLog.StructuredLogging.Json/LoggerExtensions.cs

```diff
@@ -35,7 +35,7 @@ public static void ExtendedError(this ILogger logger, string message, object log
         }
 
         public static void Extended(this ILogger logger, LogLevel logLevel, string message, object logProperties,
-            Exception ex = null)
+            Exception ex = null, string tag = "")
```

Comment:
`null` would be best IMHO, with the internals possibly changing it to `string.Empty` if it is null. You can't use `string.Empty` directly when declaring an optional parameter as it's not a constant.